python/triton/runtime/autotuner.py

```diff
@@ -111,7 +111,11 @@ def run(self, *args, **kwargs):
             for name in self.arg_names:
                 if name in all_args:
                     _args.append(all_args[name])
-            key = tuple(_args[i] for i in self.key_idx)
+            key = [_args[i] for i in self.key_idx]
```

Comment:
You also need to change the function `def get_best_config(self, *args, **kwargs)` since the way we build a `key` is changed.

Reply:
get_best_config is simplify in the way that it does not need to take in kernel arguments.